src/Runtime/Runtime/System.Windows.Controls/ComboBox.cs

```diff
@@ -163,6 +163,14 @@ protected override void OnApplyTemplate()
 
             _dropDownToggle = GetTemplateChild("DropDownToggle") as ToggleButton;
             _contentPresenter = GetTemplateChild("ContentPresenter") as ContentPresenter;
+            if (_contentPresenter != null)
+            {
+                if (_contentPresenter.IsHitTestVisible)
+                {
+                    _contentPresenter.IsHitTestVisible = false;
+                }
```

Comment:
Silverlight only sets IsHitTestVisible to false when it is using the default value
(if the user sets it to true via local value or style, we let it be)
```suggestion
                if (_contentPresenter.HasDefaultValue(IsHitTestVisibleProperty)
                {
                    _contentPresenter.IsHitTestVisible = false;
                }
```